app/views/services/autoform/index.html.erb

```diff
@@ -4,12 +4,12 @@
 
 <div class="container autoform">
   <h1><strong>Autoform</strong></h1>
-  <p class="lead">Automatické vypĺňanie kontaktných a fakturačných údajov o právnických osobách. <br><em>Vždy aktuálne údaje, rýchle a bez chýb.</em></p>
+  <p class="lead">Automatické vypĺňanie kontaktných a fakturačných údajov o právnických osobách a SZČO. <br><em>Vždy aktuálne údaje, rýchle a bez chýb.</em></p>
 
   <hr>
 
   <h2>Ukážkové použitie</h2>
-  <p class="lead"><em>Začnite písať názov právnickej osoby alebo zadajte IČO.</em> Zvyšok sa doplní automaticky.</p>
+  <p class="lead"><em>Začnite písať názov právnickej osoby prípadne SZČO, alebo zadajte IČO.</em> Zvyšok sa doplní automaticky.</p>
```

Comment:
`názov firmy, alebo zadajte IČO`

Reply:
"firma" mi tu nesedi, skor "pravnickej osoby alebo zivnostnika"

Reply:
technicky, nie kazdy zivnostnik je firma, ci? tiez som sa chcel vynut mat 2x 'alebo' po sebe

Reply:
Ano, technicky je 'firma' nespravne, ale hore piseme ze to funguje aj na zivnostnikov a podla mna message tu je proste 'zadaj nazov alebo ICO'. Radsej by som tu bol zrozumitelny ako exaktny, ale je to len navrh.